src/api/MockNetworkInterface.test.js

```diff
@@ -0,0 +1,26 @@
+const MockNetworkInterface = require("./MockNetworkInterface");
+const schema = require("eq-author-graphql-schema/schema");
+const gql = require("graphql-tag");
+
+describe("mock network interface", () => {
+  it("should be possible to query the mock network interface", () => {
+    const api = new MockNetworkInterface(schema);
+
+    const query = gql`
+      {
+        questionnaires {
+          id
+        }
+      }
+    `;
+
+    const request = {
+      query: query,
+      variables: {}
+    };
+
+    api.query(request).then(result => {
+      expect(result.data.questionnaires).toHaveLength(2);
```

Comment:
That's just knowledge I've gained from using the mocking framework. If you don't specifically tell it what you want it always creates two array elements. Also all strings are set to "Hello World". I can change it, but really the essence of this particular test was just to prove that I can query through the `NetworkInterface` implementation. I don't necessarily care about the resulting data here. The fact that we get anything back tells me that it's been passed down to the `mockServer` implementation.